cmd/minikube/cmd/env.go

```diff
@@ -55,6 +64,19 @@ func buildDockerEnvShellOutput(envMap map[string]string) string {
 	return output
 }
 
+func buildUnsetShellOutput() string {
+	envArr := []string{"DOCKER_TLS_VERIFY", "DOCKER_HOST", "DOCKER_CERT_PATH"}
+	output := ""
+	for _, envName := range envArr {
+		output += fmt.Sprintf("unset %s\n", envName)
```

Comment:
No, other shells would not work right now.  I think supporting other shells should be a feature for another issue.